frontend/src/components/account/sign-up/SignUp.test.js

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Google LLC
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     https://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SignUp from "./SignUp";
+import { getDietaryRequirements } from "../../../utils/DietaryRequirements";
+
```

Comment:
nit: can you add a comment describing what the tests in this file are testing?

Reply:
Yup